sqlglot/dialects/spark.py

```diff
@@ -41,6 +41,19 @@ def _build_datediff(args: t.List) -> exp.Expression:
     )
 
 
+def _build_dateadd(args: t.List) -> exp.Expression:
+    expression = seq_get(args, 1)
+
+    if len(args) == 2:
+        # DATE_ADD(startDate, numDays INTEGER)
+        # https://docs.databricks.com/en/sql/language-manual/functions/date_add.html
+        return exp.TsOrDsAdd(this=seq_get(args, 0), expression=expression)
```

Comment:
Let's set `unit` to `DAY` here, see hive's [constructor](https://github.com/tobymao/sqlglot/blob/main/sqlglot/dialects/hive.py#L280-L282).